src/hooks/VoiceRecorder/index.tsx

```diff
@@ -37,6 +36,16 @@ export const VoiceRecorderProvider = (props: VoiceRecorderProps): React.ReactEle
   const [mediaRecorder, setMediaRecorder] = useState<MediaRecorder>(null);
   const [isRecordable, setIsRecordable] = useState<boolean>(false);
 
+  const browserSupportMimeType: string = useMemo((): string => {
```

Comment:
Is there a reason we wrap this block \w useMemo here? 

If there's no special reason, I guess there won't be much of advantage by using useMemo here since the operation we're doing inside of this block is not really a heavy one. There could be more cons than pros when we use useMemo (same for useCallback) 

https://kentcdodds.com/blog/usememo-and-usecallback it's a bit an old article but the main concept is still working i think :)

Reply:
yeah no need to memo this